tests/profiling/settings.js

```diff
@@ -7,8 +7,8 @@ module.exports = {
   setup: {
     resolver: getAssets,
     delimiter: ['language'],
-    cache: { enabled: true },
-    batch: { enabled: true, delay: 10, limit: 50 },
+    cache: { enabled: true,  tiers: [{ limit: 6939, ttl: 300000000 }] }, //5% of the dataset assets
+    batch: { enabled: true, delay: 5, limit: 20 },
```

Comment:
Maybe we could add some context to the delay and ttl values if we set them as a multiplication and/or constant

i.e `const fiveMinutesMicroseconds = 5 * 60 * 1000000`